tests/next_tests/integration_tests/feature_tests/iterator_tests/test_scan.py

```diff
@@ -19,6 +19,8 @@
 
 @pytest.mark.uses_index_fields
 def test_scan_in_stencil(program_processor):
+    # FIXME[#1582](tehrengruber): Remove test after scan is reworked.
+    pytest.skip("Scan inside of stencil is not supported in GTIR.")
```

Comment:
why not?

Reply:
Comment is not really correct. I used a marker and am skipping it in gtfn now.